site/src/pages/AIBridgePage/RequestLogsPage/RequestLogsRow/RequestLogsRow.tsx

```diff
@@ -62,55 +107,102 @@ export const RequestLogsRow: FC<RequestLogsRowProps> = ({ interception }) => {
 							<ChevronRightIcon className="size-icon-xs" />
 						)}
 						<span className="sr-only">({isOpen ? "Hide" : "Show more"})</span>
-						{new Date(interception.started_at).toLocaleString()}
+						{new Date(interception.started_at).toLocaleString(
+							undefined,
+							customisedDateLocale,
+						)}
+					</div>
+				</TableCell>
+				<TableCell className="w-48 max-w-48">
+					<div className="w-full min-w-0 overflow-hidden">
+						<div className="flex items-center gap-3 min-w-0">
+							<Avatar
+								fallback={interception.initiator.username}
+								src={interception.initiator.avatar_url}
+								size={"lg"}
+								className="flex-shrink-0"
+							/>
+							<div className="font-medium truncate min-w-0 flex-1 overflow-hidden">
+								{interception.initiator.name ?? interception.initiator.username}
+							</div>
+						</div>
 					</div>
 				</TableCell>
-				<TableCell>
-					<div className="flex items-center gap-3">
-						<Avatar
-							fallback={interception.initiator.username}
-							src={interception.initiator.avatar_url}
-						/>
-						<div className="font-medium">{interception.initiator.username}</div>
+				<TableCell className="min-w-0">
+					{/*
+						This looks scary, but essentially what we're doing is ensuring that the
```

Comment:
```suggestion
						This is ensuring that the
```

don't decide that people should be scared of something, it's just code 😄